ni_measurementlink_service/_internal/parameter/_message.py

```diff
@@ -0,0 +1,94 @@
+import struct
+
+from google.protobuf.internal import encoder, wire_format
+
+
+def _inner_message_encoder(field_index):
```

Comment:
You customized the signature by removing is_repeated and is_packed, which duplicates the work of `_scalar_encoder` and `_vector_encoder`.